packages/flutter/test/rendering/mock_canvas.dart

```diff
@@ -327,6 +327,65 @@ abstract class PaintPattern {
   void something(PaintPatternPredicate predicate);
 }
 
+/// Matches a [Path] that contains the given `includes` points and does not contain
```

Comment:
You should point out that by "contain", you mean that that path surrounds (or doesn't surround) the region that contains that point, and not that it actually is (or is not) a member of the set of points used to draw the path.

Reply:
i pointed to [Path.contains].